dts/bindings/pcie/endpoint/brcm,iproc-pcie-ep.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Broadcom.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Broadcom iProc PCIe EP node
+
+compatible: "brcm,iproc-pcie-ep"
+
+include: base.yaml
+
+properties:
+    reg:
+      type: array
+      required: true
+      description: register space for the PAX registers
+
+    map_lowmem_base:
```

Comment:
Based on discussion on slack, map_lowmem_base, map_lowmem_size, map_highmem_base, and map_highmem_size should really be encoded in the reg's property.

Reply:
@galak I have used `reg` property to specify PCIe outbound memory ranges now. Please have a look.